src/options.cpp

```diff
@@ -653,6 +681,16 @@ void options_manager::cOpt::setValue( int iSetIn )
     }
 }
 
+//set value
+void options_manager::cOpt::setValue( bool bSetIn )
+{
+    if (sType != "bool") {
+        debugmsg("tried to set an int value to a %s option", sType.c_str());
```

Comment:
There could be not only `int` value set, right? Maybe change message a little to mention that?